react/react1/week2/class-exercise/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React, {useState} from 'react';
+import './Counter.css';
+
+
+function App() {
+  const [count, setCount] = useState(0);
+  return (
+      <div className='counter'>
+        <h1>COUNTER</h1>
+        <button onClick={() => { setCount(count + 1) }}>Click </button>
```

Comment:
setCount isn't an object, it's a function, so it shouldn't be wrapped in curly braces  =)
```suggestion
        <button onClick={() =>  setCount(count + 1) }>Click </button>
```